src/clients/apiClient.ts

```diff
@@ -10,3 +20,17 @@ export const checkBackendConnection = async (): Promise<boolean> => {
   
   return response.ok;
 }
+
+export async function createSighting(newSighting: NewSighting) {
```

Comment:
Can we return a promise here?
```suggestion
export async function createSighting(newSighting: NewSighting): Promise<Response> {
```